gdx-video-gwt/src/com/badlogic/gdx/video/VideoPlayerGwt.java

```diff
@@ -90,7 +97,7 @@ public void pause () {
 
 	@Override
 	public void resume () {
-		if (v != null && (v.getCurrentTime() == 0 || v.getCurrentTime() < v.getDuration())) v.play();
```

Comment:
I initially wrote: "We'll have to confirm if this check was important (may have prevented `resume()` from restarting an ended video)."

But then I saw what I had wrote at #74 about removing this. Go for it. We've reached the limit of how far back I can remember things.

Reply:
Hmm yeah. My thought about a resume function is that it is pretty pointless if we have a play function that starts playing the video... So it has pretty much the same functionality. That's why I added `@Deprecated`. Also, when I think about pressing _play_ on a video player, I would expect it to restart a video when it is finished. What do you think?

Reply:
I'd expect "play" in a UI to restart if finished. Even many VCRs had auto-rewind. It's only in programmatic use that caution has to be applied, especially regarding any platform differences, so as to avoid accidental restarts. But `Music` doesn't have a `resume()` so it's probably fine to scrap.

Reply:
Yes, I would adjust the other backends to restart as well

Reply:
Everything should behave now like this, but I can't test iOS